### PR TITLE
Feature homepage

### PR DESCRIPTION
**Pull request by appiersign**

### CODE REVIEW

**Comment by duducus** [comment]
Good structure, all html semantics is in place. 
the tags can be more descriptive

File: style.css [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+  width: 305px;
+  min-height: 200px;
+  float: right;
+  position: relative;
+}
+#up-next {
+margin: 0;
+}
+#recommendations {
+}
```
Comment (pelzolga123) [inline style.css]: Empty selector should be deleted.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+                                <li><a id="subscribe-btn"><i class="fab
+                                            fa-youtube"></i>Subscribe </a></li>
+                            </ul>
+                        </div>
+
+                        <hr style="width: 95%;">
+
+                        <div class="mt-15" style="width: 95%;">
+                            <ul class="no-list-style float-left"
+                                style="margin-left: -180px">
```
Comment (pelzolga123) [inline index.html]: No inline styling. Please, move all styling to css.

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project needs corrections before it can be approved.

**Comment by pelzolga123** [comment]
## Hi Team!  
### Your project looks :+1: good!
But there are few things that need to be corrected:
#### General
* [x] Check HTML markup with this [validator](https://validator.w3.org/#validate_by_upload). Now it shows errors in your project.
* [x] README.md file is not descriptive.

#### Navbar
* [x] The search bar should be aligned to the middle of the navbar.  [Link to screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/searchbar_align_requirement.png) 
* [x] It should have at least three embedded images on the rightmost part of the navbar that represent icons and/or an avatar. [Link to screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/icons_requirement.png) 

#### Video section
* [x] It should show the number of views. [Link to screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_views_requirement.png)
* [x] 'Share' should be a button. [Link to screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_share_requirement.png)
* [x] 'Subscribe' should be a button. [Link to screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/video_subscribe_requirement.png)

###  Please kindly make this changes and send another code review request.

If you have any questions or need help, you can message me on slack. *Happy coding*

**Comment by Anna-Myzukina** [comment]
### Pull Request under the review by Anna Muzykina (TSE)

**Comment by Anna-Myzukina** [comment]
### Hi team!

### Nice job! Project looks! :+1: 

The readme file looks good. I hope these links can help make it more professional in the next projects [Professional Github repository](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) and this [Markdown Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet).

### But first of all let's fix the few things and we can move on.

#### Page structure

##### Navigation bar steel has few things to change

* [x] The search bar should be is aligned to the middle of the navbar 


* [x] It should has at least three embedded images on the rightmost part of the navbar that represent icons and/or an avatar 

As we can see, now in your project the search bar still isn't aligned to the middle of the navbar and three images are on the leftmost part, but they should be on the rightmost part.
header


* Please remember that your branch name should be descriptive (e.g., refactor-authentication, user-content-cache-key, make-retina-avatars), so that others can see what is being worked on. You can read more about it [here](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/). I hope in the next project you can do this!

### Make changes and submit another code review request.

*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*

**Review by Anna-Myzukina — CHANGES_REQUESTED** [review]
Kindly recommended make next changes: the search bar still isn't aligned to the middle of the navbar and three images are on the leftmost part, but they should be on the rightmost part.

**Comment by shshamim63** [comment]
## Status :recycle: [!! Required Changes !!] :recycle: 
___
Hi @appiersign  & @MotivateJ  :wave: ,
You guys have gathered all the items that are actually needed to decorate the page. There are no  HTML markup error. In one word great job :clap: . But there are some issues that are actually blocking me to approve this project. And i am sure that you guys will fix them easily :sunglasses: .
___
##### Please follow the instructions below :point_down: to declare this project as complete.
___
### :sparkle: [Mandatory Changes] :space_invader: 
___
1. The items on the navbar are not displaying as it should. Is because of declaring with using px. You can look at this [link](https://kyleschaeffer.com/css-font-size-em-vs-px-vs-pt-vs-percent). It will give you clear idea about the units of css properties.
image
 This is the view i am getting. Now how can I mimic the environment of the reviewer. Well I use [this](https://responsivedesignchecker.com/) one. So please rearrange the item.
2. The aside items are also breaking their flow.
image
If you inspect it, you will get the cause. Fix them too.
3. I can see some indentation problem. For an example, <li> open and closing tag should be vertically aligned.
https://github.com/appiersign/youtube-homepage/blob/fbf245a1935b978602403625388282d441ca3150/index.html#L135-L137

___

### :clubs: [Optional Changes] :clubs: 
___
1. You could use a reset css to reset your browsers default stylesheet. Check this [link](https://stackoverflow.com/questions/11578819/css-reset-what-exactly-does-it-do) for a basic understanding. 
2. There are couple of repetitive codes in css. For an example `width: auto;`. Maybe you can create a class for this and you can use that whenever it is needed.
3. You can make your folder more organized by creating a folder called asstes. Inside that you can create sub folders for different files (image, style(for css)).
4. I think link should appear after all other tags inside head tag.
https://github.com/appiersign/youtube-homepage/blob/fbf245a1935b978602403625388282d441ca3150/index.html#L3-L12
___
#### Please fix the above mentioned issue and send another review request.
___
## :herb: Happy Coding:
### Shakhawat Hossain
[Linkedin](https://www.linkedin.com/in/shakhawathossainshamim/) | [Slack](Shakhawat Hossain)

**Review by pelzolga123 — APPROVED** [review]
Approved.

**Comment by pelzolga123** [comment]
## Great  work, Congratulation :+1: Project Approved.
### You did all changes required previously. Go ahead and merge to master.
_Goodluck in your next project_. *Happy coding*

**Pull request merged**